src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -81,8 +82,8 @@ public function normalize($exception, $format = null, array $context = [])
         return $data;
     }
 
-    public function supportsNormalization($data, $format = null)
+    public function supportsNormalization($data, string $format = null, array $context = [])
```

Comment:
```suggestion
    public function supportsNormalization($data, $format = null, array $context = [])
```

For backwards compatibility to Symfony 4.4 its required not to set the string typehint.